synapse/push/push_rule_evaluator.py

```diff
@@ -145,7 +170,13 @@ def matches(
         else:
             return True
 
-    def _event_match(self, condition: dict, user_id: str) -> bool:
+    def _event_match(
+        self,
+        condition: dict,
+        user_id: str,
+        event: EventBase,
+        event_value_cache: Dict[str, str],
+    ) -> bool:
         pattern = condition.get("pattern", None)
```

Comment:
Since you're changing this can you add a docstring please? Doesn't need to be super detailed, just what the function does, roughly what the arguments are and what it returns